src/ldf-client-ui.js

```diff
@@ -734,11 +735,10 @@ require('leaflet/dist/images/marker-shadow.png');
             geoFeature.properties.name = valueLabel.split('@', 1)[0].replace(/['"]+/g, '');
 
           // Add feature to map
-          L.geoJSON(geoFeature, {
+          let laayer = L.geoJSON(geoFeature, {
```

Comment:
Typo

Reply:
Sorry about this , needs to be mapLayer, should change it in my  next commit. Need a word that is not just layer such that it cannot conflict with the geoJSON features layers object